test/EVMHost.cpp

```diff
@@ -115,6 +124,17 @@ void EVMHost::selfdestruct(const evmc::address& _addr, const evmc::address& _ben
 
 evmc::result EVMHost::call(evmc_message const& _message) noexcept
 {
+	// Force failure for Byzantium precompiles on pre-Byzantium chains.
+	// NOTE: >=5 && <=8 would be easier to read, but those overloads don't exist in EVMC
+	if (
+		_message.destination > 0x0000000000000000000000000000000000000004_address &&
```

Comment:
```suggestion
		0x0000000000000000000000000000000000000004_address < _message.destination &&
```
Then we don't need the operator...